src/proto/ctl/storage.proto

```diff
@@ -22,7 +22,8 @@ message StorageScanReq {
 	ScanScmReq scm = 2;
 }
 
-message MemInfo {
+message SysMemInfo
```

Comment:
This name change at the protobuf level probably isn't necessary. Wouldn't block on this though.